dist/obs-service-tar_scm.spec

```diff
@@ -22,7 +22,7 @@
 %bcond_with    obs_scm_testsuite
 %endif
 
-%if 0%{?suse_version} >= 1500 || 0%{?fedora_version} >= 29
+%if 0%{?suse_version} >= 1500 || 0%{?fedora_version} >= 29 || 0%{?sle_version} >= 120000
```

Comment:
%if 0%{?suse_version} >= 1315 || 0%{?fedora_version} >= 29

should do the same and include leap 42.x